### ISSUE TITLE
The default dmesg whitelist should be updated for the new network interface name style

### ISSUE THREAD

**Issue by fsfod** [softdevteam/krun#363]
Theses are the the messages I'm seeing on bencher3.
```
[ 4.873919] r8169 0000:06:00.0 enp6s0: link up
[ 377.233591] random: crng init done
```

https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/
> Names incorporating Firmware/BIOS provided index numbers for on-board devices (example: eno1)
> Names incorporating Firmware/BIOS provided PCI Express hotplug slot index numbers (example: ens1)
> Names incorporating physical/geographical location of the connector of the hardware (example: enp2s0)
> Names incorporating the interfaces's MAC address (example: enx78e7d1ea46da)

**Comment by ltratt** [comment]
It's not totally clear to me whether the default list should include this or whether users should be encouraged to add it to their config files. I think i'm inclined towards the latter, but can be persuaded the other way.

**Comment by fsfod** [comment]
The default list already filters out the old "eth" variant and has this comment.
# Bringing the network up and down on Linux (which some experiments may wish to do) makes some noise. Ignore.

**Comment by ltratt** [comment]
Yes, but in a sense that's its weakness: we hardcoded in something which Linux has now changed. That said, given that we allow users to override the whitelist, updating the defaults is probably safe enough.

**Comment by vext01** [comment]
We just need too make the regex more general to match any driver name.

**Comment by ltratt** [comment]
I still think I'm inclined towards saying this should be wholly in the user's config file. If so, all we'd have to do is delete the regex's from the main code!
